### PR TITLE
Screen Add

### PR DESCRIPTION
**Pull request by CrypeDJ**

### CODE REVIEW

File: app/src/main/java/com/crype/bankapp/presentation/components/EnterButton.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import androidx.compose.ui.unit.sp
+import com.crype.bankapp.ui.theme.Blue
+import com.crype.bankapp.ui.theme.BluePressed
+import com.crype.bankingapp.ui.theme.Typography
+
+@Composable
+fun EnterButton (
+    label: String,
+    //onClick: () ->Unit
+){
```
Comment (Razor0744) [inline app/src/main/java/com/crype/bankapp/presentation/components/EnterButton.kt:21]: unused code

File: app/src/main/java/com/crype/bankapp/presentation/components/TopBar.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import androidx.compose.ui.unit.sp
+import com.crype.bankapp.R
+import com.crype.bankingapp.ui.theme.Typography
+
+@Composable
+fun TopBar(
+    //onBackClick:() -> Unit,
+    //onFilterClick:() -> Unit,
+    title: String
+) {
```
Comment (Razor0744) [inline app/src/main/java/com/crype/bankapp/presentation/components/TopBar.kt:24]: the same problem

File: app/src/main/java/com/crype/bankapp/presentation/screen/AddScreen.kt [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                    EnterField(
+                        title = fields[it],
+                        keyboardType = when (fields[it]) {
+                            "Amount", "Date" -> KeyboardType.Number
+                            else -> KeyboardType.Text
+                        }
+                    )
+
+
+                }
```
Comment (Razor0744) [inline app/src/main/java/com/crype/bankapp/presentation/screen/AddScreen.kt:66]: empty lines

File: app/src/main/java/com/crype/bankapp/presentation/screen/TransactionInfoScreen.kt [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                modifier = Modifier.padding(vertical = 40.dp)
+            )
+            Column(
+                modifier = Modifier.imePadding()
+            ) {
+                val transactionModel = TransactionModel()
+                fields.forEach { label ->
+
+                    TextField(
+                        title = label,
```
Comment (Razor0744) [inline app/src/main/java/com/crype/bankapp/presentation/screen/TransactionInfoScreen.kt:59]: empty line

File: app/src/main/java/com/crype/bankapp/presentation/components/EnterButton.kt (hunk 1 above)
Comment (CrypeDJ) [reply app/src/main/java/com/crype/bankapp/presentation/components/EnterButton.kt:21]: got it

File: app/src/main/java/com/crype/bankapp/presentation/components/TopBar.kt (hunk 2 above)
Comment (CrypeDJ) [reply app/src/main/java/com/crype/bankapp/presentation/components/TopBar.kt:24]: got it

File: app/src/main/java/com/crype/bankapp/presentation/screen/TransactionInfoScreen.kt (hunk 4 above)
Comment (CrypeDJ) [reply app/src/main/java/com/crype/bankapp/presentation/screen/TransactionInfoScreen.kt:59]: got it

File: app/src/main/java/com/crype/bankapp/presentation/screen/AddScreen.kt (hunk 3 above)
Comment (CrypeDJ) [reply app/src/main/java/com/crype/bankapp/presentation/screen/AddScreen.kt:66]: got it

**Pull request merged**